OpenRA.Game/Graphics/Sheet.cs

```diff
@@ -143,6 +143,10 @@ public void ReleaseBuffer()
 
 		public bool ReleaseBufferAndTryTransferTo(Sheet destination)
 		{
+			// If the renderer is null, it means we aren't commiting data to the GPU, so let's not delete our data.
+			if (Game.Renderer == null)
```

Comment:
This should be moved. After the `ReleaseBuffer()` call. Before the transfer of the buffer to the destination.

Reply:
that also works